README.md

```diff
@@ -253,7 +253,7 @@ from mobly.controllerse import android_device
 class HelloWorldTest(base_test.BaseTestClass):
 
   def setup_class(self):
-    self.ads = self.register_controller(android_device)
+    self.ads = self.register_controller(android_device, min_number=2)
```

Comment:
If they don't know what register_controller does at this point in README, we have bigger problems?...
Looks like we need more inline comments in general.